src/SSHDebugTests/SSHConnectionStringTests.cs

```diff
@@ -45,9 +46,9 @@ public void IPv6ConnectionStrings()
             ipv6TestStrings.Add(
                 new ConnectionStringTestItem()
                 {
-                    // valid with custom username
+                    // valid with username:password
```

Comment:
>:password [](http://example.com/codeflow?start=42&length=9)

I think you want to restore this comment

Reply:
Isn't the new comment more explicit about what is being tested with this case?

Reply:
Right, but didn't you change that test case so it does NOT have a password?

Reply:
Oh, I see. That test case is weird. I wonder if we should get rid of it in favor of your new one?

Reply:
Never mind. I see that it is good as an IPv6 test. I wonder what in the world it was trying to test before...